FlightsWebsite/src/js/flights.js

```diff
@@ -26,4 +124,174 @@ function status(response) {
 
 function json(response) {
     return response.json()
-}
\ No newline at end of file
+}
+
+function showTotalFlights(data) {
+
+    let chart = new CanvasJS.Chart("chartContainer", {
+        animationEnabled: true,
+        theme: "light2", // "light1", "light2", "dark1", "dark2"
+        title: {
+            text: "Total number of flights per month"
+        },
+        axisY: {
+            title: "Total number of flights",
+            interval: 5000
+        },
+        axisX: {
+            interval: 1
+        },
+        data: [{
+            type: "column",
+            showInLegend: true,
+            legendMarkerColor: "grey",
+            legendText: "Month",
+            dataPoints: data
+        }]
+    });
+    chart.render();
+}
+
+function showTopDestinations(data) {
+
+    let chart = new CanvasJS.Chart("chartContainer", {
+        theme: "light2", // "light1", "light2", "dark1", "dark2"
+        exportEnabled: true,
+        animationEnabled: true,
+        title: {
+            text: "Top 10 Destinations"
+        },
+        data: [{
+            type: "pie",
+            startAngle: 25,
+            toolTipContent: "<b>{label}</b> : {y} flights",
+            showInLegend: "true",
+            legendText: "{label}",
+            indexLabelFontSize: 16,
+            indexLabel: "{label}",
+            dataPoints: data
+        }]
+    });
+    chart.render();
+}
+
+function showTotalFlightsFromOrigins(dataLGA, dataJFK, dataEWR) {
```

Comment:
I don't think that this is a correct chart.. The requirement says: "Total number of flights per month from the three origins in one plot. We want to be ableto see
-Frequency
-Frequency stacked
-Stacked percentage"

You have to show all of those types of data in one chart. Perhaps, line chart is not the best for it. Try stacked column chart

Reply:
I would not say all of those types in one chart .. check the examples provided by Richard .. there are those 3 types of graphs(Frequency,Frequency stacked and Stacked percentage) in 3 separate graphs .. so imo we should display 3 graphs below each other at this page .. or make like carousel or smth